client/package.json

```diff
@@ -824,7 +824,7 @@
     "test:grammar": "vscode-tmgrammar-test ./test/grammars/test-cases/*.bb",
     "snap-grammar": "vscode-tmgrammar-snap ./test/grammars/snaps/*.bb -u",
     "vscode:prepublish": "npm install --omit=dev && cd ../server && npm install --omit=dev && npm run installServer",
-    "package": "npm run vscode:prepublish && vsce package --target linux-x64 && vsce package --target linux-arm64 && vsce package --target linux-armhf && vsce package --target win32-x64 && vsce package --target win32-arm64 && vsce package --target web"
+    "package": "npm run vscode:prepublish && vsce package --target linux-x64 && vsce package --target linux-arm64 && vsce package --target linux-armhf && vsce package --target win32-x64 && vsce package --target win32-arm64 && vsce package --target web && vsce package --target darwin-x64 && vsce package --target darwin-arm64 && vsce package --target alpine-x64 && vsce package --target alpine-arm64"
```

Comment:
Actually let's just get back to universal now? the contents are the same. Do we need a specific bundle for web? Probably not.

Reply:
No, the vscode only needs to know the value in `browser` property. We could revert it back to the package for all. But what about the platforms that are not here? Such as `win32-ia32`. Though, I think the safer way is to put the ones that we know it works.

Reply:
At this point I think we can consider everythin to work.

Reply:
I suggest going back to a universal package. Make sure it works on web if possible. Update the packaging and deployment script if required.